oracledb-native/src/main/java/org/ballerinalang/oracledb/Constants.java

```diff
@@ -0,0 +1,179 @@
+
+    /**
+     * Constants related to Oracle DB types supported.
+     */
+    public static final class Types {
+
+        /**
+         * Constants related to Oracle Database type names.
+         */
+        public static final class OracleDbTypes {
+            public static final String INTERVAL_YEAR_TO_MONTH = "INTERVAL_YEAR_TO_MONTH";
+            public static final String INTERVAL_DAY_TO_SECOND = "INTERVAL_DAY_TO_SECOND";
+            public static final String OBJECT_TYPE = "OBJECT";
+            public static final String VARRAY = "VARRAY";
+            public static final String NESTED_TABLE = "NESTED_TABLE";
+        }
+
+        /**
+         * Constants related to custom ballerina type names.
+         */
+        public static final class CustomTypes {
+            public static final String INTERVAL_YEAR_TO_MONTH = "IntervalYearToMonthValue";
+            public static final String INTERVAL_DAY_TO_SECOND = "IntervalDayToSecondValue";
+            public static final String OBJECT = "ObjectTypeValue";
+            public static final String VARRAY = "VarrayValue";
+            public static final String NESTED_TABLE = "CustomTableValue";
+            public static final String XML = "XmlValue";
+        }
+
+        /**
+         * Constants related to the attributes of INTERVAL_YEAR_TO_MONTH Oracle DB type.
+         */
+        public static final class IntervalYearToMonth {
+            public static final String YEARS = "years";
+            public static final String MONTHS = "months";
+
+        }
+
+        /**
+         * Constants related to the attributes of INTERVAL_DAY_TO_SECOND Oracle DB type.
+         */
+        public static final class IntervalDayToSecond {
+            public static final String DAYS = "days";
+            public static final String HOURS = "hours";
+            public static final String MINUTES = "minutes";
+            public static final String SECONDS = "seconds";
+        }
+
+        /**
+         * Constants related to the attributes of OBJECT Oracle DB type.
+         */
+        public static final class OracleObject {
+            public static final String TYPE_NAME = "typename";
+            public static final String ATTRIBUTES = "attributes";
+        }
+
+        /**
+         * Constants related to the attributes of VARRAY Oracle DB type.
+         */
+        public static final class Varray {
+            public static final String NAME = "name";
+            public static final String ELEMENTS = "elements";
+        }
+
+        /**
+         * Constants related to the attributes of XML Oracle DB type.
+         */
+        public static class Xml {
+            public static final String XML = "xml";
+        }
+    }
+
+    public static final String DRIVER = "jdbc:oracle:thin:@//";
+    public static final String ORACLE_DATASOURCE_NAME = "oracle.jdbc.pool.OracleDataSource";
+    public static final String CUSTOM_RESULT_ITERATOR_OBJECT = "CustomResultIterator";
+}
+
```

Comment:
Extra new line

Reply:
Check all the files

Reply:
fixed with 25bf1c712147fcb7da97ffecf761acbaf0324856